lib/runtime/wasmer/exports.go

```diff
@@ -135,11 +135,15 @@ func (in *Instance) ExecuteBlock(block *types.Block) ([]byte, error) {
 
 	// remove seal digest only
 	for _, d := range block.Header.Digest.Types {
-		switch d.Value().(type) {
+		digestValue, err := d.Value()
+		if err != nil {
+			return nil, err
```

Comment:
nit (also adding the digest type index to the error context would be nice)

```suggestion
			return nil, fmt.Errorf("getting digest type value: %w", err)
```